nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
> Hm, I actually don't see how passing the scrubbed pkgs can work. doc/manual expects a real pkgs, since it uses various packages (stdenv, libxslt, ...).

That's because I only scrub `pkgs` used to generate the options, not `pkgs` actually used by the manual generator itself :-)